### ISSUE TITLE
File Symbols not working

### ISSUE THREAD

**Issue by nickvanderstel** [atom/symbols-view#69]
I have my tags file in the root of my project and can bring up the project symbol list which works fine. When I try to bring up the file symbol list, there is nothing in the list. There is nothing showing up in the console so I'm not quite sure where to dig next.

Running v0.175 on Linux.

**Comment by kevinsawicki** [comment]
What file do you have open when the file symbol list is empty?

**Comment by nickvanderstel** [comment]
A C source file.

**Comment by kevinsawicki** [comment]
Does it happen for all c files or only certain ones?

Can you share one of the files it occurs for?

**Comment by nickvanderstel** [comment]
I will do that, but it will probably have to wait until the morning (machine is at work right now).

**Comment by nickvanderstel** [comment]
It doesn't work even with a file as simple as this:

```
int foo(void);

int main(void)
{
    int ret;

    ret = foo();

    return ret;
}

int foo(void)
{
    int x;

    x = 4;

    return x;
}
```

I created a tags file for this and project symbol search works fine.

**Comment by nickvanderstel** [comment]
It looks like the problem is that symbols-view is using its own ctags binary that is 64-bit (vendor/ctags-linux). The machine having the problem is a 32-bit install.

**Comment by nelson6e65** [comment]
I have the same issue. :confused: 

@nickvanderstel How can I replace that?

**Comment by nickvanderstel** [comment]
Copy /usr/bin/ctags to /local/share/atom/resources/app/node_modules/symbols-view/vendor/ctags-linux.

**Comment by caleb** [comment]
I'm on a OS X 10.10 system, running Atom 1.0.11 and I'm noticing the same thing. I could swear this worked just a couple days ago since I was playing with my ctags generating script, but I could be wrong.

I can use the project symbol finder just fine, but I get "No symbols found" when using CMD+R to find symbols in a file.

I tried removing all of my plugins and the same thing happens.

**Comment by lee-dohm** [comment]
@caleb, if I recall correctly, OS X hasn't had a 32-bit kernel in at least the last couple major releases. So your issue shouldn't be for the same root cause as mentioned here.

**Comment by caleb** [comment]
Yeah, this is a 64-bit system, but it was the same thing so I thought I'd mention it since this issue wasn't marked as closed :)

Should I open a new issue?

**Comment by lee-dohm** [comment]
Yes, I would suggest opening a new issue. Also include as much information as you can, especially the information that @kevinsawicki asked for up top.
